.yarnrc.yml

```diff
@@ -4,4 +4,4 @@ plugins:
     - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
       spec: '@yarnpkg/plugin-interactive-tools'
 
-yarnPath: .yarn/releases/yarn-berry.cjs
+yarnPath: .yarn/releases/yarn-3.2.4.cjs
```

Comment:
unrelated changes. Needed to update yarn version, because the version we used in mistica has a bug with the cache when switching between node versions